inc/Engine/Optimization/RUCSS/Database/Tables/UsedCSS.php

```diff
@@ -60,21 +61,22 @@ public function __construct() {
 	 */
 	protected function set_schema() {
 		$this->schema = "
-			id               bigint(20) unsigned NOT NULL AUTO_INCREMENT,
-			url              varchar(2000)       NOT NULL default '',
-			css              longtext                     default NULL,
-			hash             varchar(32)                  default '',
-			error_code       varchar(32)             NULL default NULL,
-			error_message    longtext                NULL default NULL,
-			unprocessedcss   longtext                NULL,
-			retries          tinyint(1)          NOT NULL default 1,
-			is_mobile        tinyint(1)          NOT NULL default 0,
-			job_id           varchar(255)        NOT NULL default '',
-			queue_name       varchar(255)        NOT NULL default '',
-			status           varchar(255)        NOT NULL default '',
-			modified         timestamp           NOT NULL default '0000-00-00 00:00:00',
-			last_accessed    timestamp           NOT NULL default '0000-00-00 00:00:00',
-			submitted_at     timestamp           NULL,
+			id               		bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+			url              		varchar(2000)       NOT NULL default '',
+			css              		longtext                     default NULL,
+			hash             		varchar(32)                  default '',
+			error_code       		varchar(32)             NULL default NULL,
+			error_message    		longtext                NULL default NULL,
+			unprocessedcss   		longtext                NULL,
+			retries          		tinyint(1)          NOT NULL default 1,
+			is_mobile        		tinyint(1)          NOT NULL default 0,
+			job_id           		varchar(255)        NOT NULL default '',
+			queue_name       		varchar(255)        NOT NULL default '',
+			status           		varchar(255)        NOT NULL default '',
+			modified         		timestamp           NOT NULL default '0000-00-00 00:00:00',
+			last_accessed    		timestamp           NOT NULL default '0000-00-00 00:00:00',
+			submitted_at     		timestamp           NULL,
+			not_proceed_before     	timestamp           NULL,
```

Comment:
`not_proceed_before` or `not_processed_before` ?

Reply:
It's been changed to `next_retry_time`